cfme/fixtures/automate.py

```diff
@@ -100,7 +101,8 @@ def test_foo(import_datastore, import_data):
         import_type="file", file_path=file_path
     )
     domain = datastore.import_domain_from(import_data.from_domain, import_data.to_domain)
-
+    assert domain.exists
+    with update(domain):
```

Comment:
the `cfme.automate.explorer.domain.Domain` class needs to be `Updateable` for this to work properly for comparison.  Looks like a few of the Automate entity classes are not Updatable but have an update method, and are using this context manager.